src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <section class="calendar calendar--monthLength-30">
```

Comment:
Ideally we would like the whole calendar to control the starting day of the month. Also remember about BEM naming convention:
```suggestion
    <section class="calendar calendar__start-day--tue calendar__month-length--28">
```